examples/nidaqmx_analog_input/pyproject.toml

```diff
@@ -7,7 +7,7 @@ authors = ["National Instruments"]
 [tool.poetry.dependencies]
 python = "^3.8"
 nidaqmx = ">=0.6.3"
-ni-measurementlink-service = "*"
+ni-measurementlink-service = "1.0.0"
```

Comment:
Let's specify "^1.0.0" so that the examples use the latest 1.x.

Later, when we update an example to use a new feature that is not supported in 1.0.0, we should update the example to use "^1.1" or whatever it needs.

If we have to make compatibility-breaking changes, then we should update ni-measurementlink-service to 2.x to signal that there was a compatibility break. Clients that request "^1.0.0" will keep using the latest 1.x.

(This is basically semantic versioning.)